libraries/chain/include/eos/chain/rate_limiting_object.hpp

```diff
@@ -0,0 +1,43 @@
+/**
+ *  @file
+ *  @copyright defined in eos/LICENSE.txt
+ */
+#pragma once
+#include <eos/chain/types.hpp>
+#include <eos/chain/authority.hpp>
+
+#include "multi_index_includes.hpp"
+
+namespace eos { namespace chain {
+
+   class rate_limiting_object : public chainbase::object<rate_limiting_object_type, rate_limiting_object> {
+//      OBJECT_CTOR(rate_limiting_object)
```

Comment:
Why are you not using the macro? It looks like your code is just doing what the macros does.

Reply:
Was having an error and was putting in print statements and forgot to clean up, will do that now